Framework/Algorithms/src/UnwrapMonitorsInTOF.cpp

```diff
@@ -331,15 +331,27 @@ void UnwrapMonitorsInTOF::exec() {
   const auto workspaceIndices =
       getWorkspaceIndicesForMonitors(outputWorkspace.get());
 
+  bool spectrumIsHistogramData(true);
```

Comment:
Is there a reason this is defined outside of the loop? It would seem best to define it locally in the loop.